apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.tsx

```diff
@@ -8,6 +11,15 @@ import { AssessmentInstanceRow } from '../../src/pages/instructorAssessmentInsta
 
 import { getPopoverTriggerForContainer } from './lib/popover.js';
 
+type TimeLimitPlusMinus =
```

Comment:
Sounds good. I updated this to `TimeLimitAction` and now use `action` instead of `plusMinus` throughout. However, is this too similar to `__action` that we are already using in the form?

Reply:
Yeah, that's unfortunate. It's probably just find. Do you have any suggestions for another word we could use here?